### ISSUE TITLE
Parts of raw SQL response get lost

### ISSUE THREAD

**Issue by bcat-eu** [manticoresoftware/manticoresearch-php#56]
To reproduce:

```
<?php
require_once __DIR__ . '/vendor/autoload.php';
$config = ['host' => '127.0.0.1', 'port' => 9308];
$client = new \Manticoresearch\Client($config);

$index = new \Manticoresearch\Index($client);
$index->setName('test');
$index->create(
    ['word' => ['type' => 'text']],
    [
        'min_infix_len' => '3',
        'index_exact_words' => '1',
        'expand_keywords' => '1',
        'bigram_index' => 'all'
    ],
);

$index->addDocuments([['id' => 1, 'word' => 'one two three'], ['id' => 2, 'word' => 'one five seven'], ['id' => 3, 'word' => 'zero one six']]);

$query = "CALL KEYWORDS('on*', 'test', 1 as stats, 'hits' as sort_mode);";
$params = ['mode' => 'raw', 'body' => ['query' => $query]];

print_r($client->sql($params));
```

That will deliver the last result and lose the rest, something like that:

```
Array
(
    [1] => Array
        (
            [tokenized] => on*
            [normalized] => one two
            [docs] => 1
            [hits] => 1
        )

)
```

The underlying response would have all the data though:

```
array(5) {
  ["columns"]=>
  array(5) {
    [0]=>
    array(1) {
      ["qpos"]=>
      array(1) {
        ["type"]=>
        string(6) "string"
      }
    }
    [1]=>
    array(1) {
      ["tokenized"]=>
      array(1) {
        ["type"]=>
        string(6) "string"
      }
    }
    [2]=>
    array(1) {
      ["normalized"]=>
      array(1) {
        ["type"]=>
        string(6) "string"
      }
    }
    [3]=>
    array(1) {
      ["docs"]=>
      array(1) {
        ["type"]=>
        string(6) "string"
      }
    }
    [4]=>
    array(1) {
      ["hits"]=>
      array(1) {
        ["type"]=>
        string(6) "string"
      }
    }
  }
  ["data"]=>
  array(4) {
    [0]=>
    array(5) {
      ["qpos"]=>
      string(1) "1"
      ["tokenized"]=>
      string(3) "on*"
      ["normalized"]=>
      string(3) "one"
      ["docs"]=>
      string(1) "3"
      ["hits"]=>
      string(1) "3"
    }
    [1]=>
    array(5) {
      ["qpos"]=>
      string(1) "1"
      ["tokenized"]=>
      string(3) "on*"
      ["normalized"]=>
      string(8) "one five"
      ["docs"]=>
      string(1) "1"
      ["hits"]=>
      string(1) "1"
    }
    [2]=>
    array(5) {
      ["qpos"]=>
      string(1) "1"
      ["tokenized"]=>
      string(3) "on*"
      ["normalized"]=>
      string(7) "one six"
      ["docs"]=>
      string(1) "1"
      ["hits"]=>
      string(1) "1"
    }
    [3]=>
    array(5) {
      ["qpos"]=>
      string(1) "1"
      ["tokenized"]=>
      string(3) "on*"
      ["normalized"]=>
      string(7) "one two"
      ["docs"]=>
      string(1) "1"
      ["hits"]=>
      string(1) "1"
    }
  }
  ["total"]=>
  int(0)
  ["error"]=>
  string(0) ""
  ["warning"]=>
  string(0) ""
}
```

It seems that the SqlToArray class makes too many assumptions about the structure of the input data and in case of the above response this line https://github.com/manticoresoftware/manticoresearch-php/blob/0e56d0e4c6bcb28035b52122fb2b943710d131b7/src/Manticoresearch/Response/SqlToArray.php#L19 will keep overwriting the relevant part of data so only the last result remains.

**Comment by githubmanticore** [comment]
Fixed #56   more changes

Fixed #56; improved sql emulated commands, output format for some of them slightly changed; fixed composer requirements, so they don't conflict one with another; fixed tests via commit https://github.com/manticoresoftware/manticoresearch-php/commit/e929c743940e6eb3104fb623854a983881203950

**Comment by githubmanticore** [comment]
Fixed #56   more changes

Fixed #56; improved sql emulated commands, output format for some of them slightly changed; fixed composer requirements, so they don't conflict one with another; fixed tests via commit https://github.com/manticoresoftware/manticoresearch-php/commit/e929c743940e6eb3104fb623854a983881203950

**Comment by sanikolaev** [comment]
The fix is available in v1.6.2
